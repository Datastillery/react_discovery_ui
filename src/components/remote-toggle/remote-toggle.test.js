import React from 'react'
import { shallow } from 'enzyme'
import RemoteToggle from './remote-toggle'

describe('<RemoteToggle />', () => {
  test('renders', () => {
    const wrapper = shallow(<RemoteToggle />)
    expect(wrapper).toMatchSnapshot()
  })
})
