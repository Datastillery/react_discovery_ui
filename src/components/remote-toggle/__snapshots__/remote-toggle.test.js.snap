// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RemoteToggle /> renders 1`] = `ShallowWrapper {}`;
